charts/argo-workflows/Chart.yaml

```diff
@@ -18,3 +18,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: added
       description: add imagePullSecret for workflow sa
+    - kind: updated 
```

Comment:
Probably `updated` is not supported in `artifacthub.io/changes` , so maybe `fixed` ?

Ref: https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog

<img width="987" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/5c288e15-fff9-43dd-9143-fdb3cf324d24">